src/System.Net.Http/ref/System.Net.Http.cs

```diff
@@ -85,6 +85,7 @@ public partial class HttpClientHandler : System.Net.Http.HttpMessageHandler
         public System.Security.Cryptography.X509Certificates.X509CertificateCollection ClientCertificates { get { throw null; } }
         public System.Net.CookieContainer CookieContainer { get { throw null; } set { } }
         public System.Net.ICredentials Credentials { get { throw null; } set { } }
+        public static System.Func<System.Net.Http.HttpRequestMessage, System.Security.Cryptography.X509Certificates.X509Certificate2, System.Security.Cryptography.X509Certificates.X509Chain, System.Net.Security.SslPolicyErrors, bool> DangerousAcceptAnyServerCertificateValidator { get { throw null; } }
```

Comment:
We don't need to add ifdefs here to limit to core right? (I remember a time when we used to do that in ref but I'm not yet caught up with recent changes.)